sprocs/questions_select.sql

```diff
@@ -1,15 +1,44 @@
-CREATE FUNCTION
+DROP FUNCTION IF EXISTS questions_select(bigint);
+DROP FUNCTION IF EXISTS questions_select(bigint, bigint);
+
+CREATE OR REPLACE FUNCTION
```

Comment:
`DROP FUNCTION` is no longer necessary based on recent changes.

```suggestion
CREATE FUNCTION
```

Reply:
Sorry, should this be `CREATE FUNCTION` or `CREATE OR REPLACE FUNCTION`? My DROP statements were based on an older version of the file and I wasn't sure how to incorporate the changes.

Reply:
No need for replace in new versions, see other sprocs in master.